minitests/timing/runme.tcl

```diff
@@ -1,95 +1,4 @@
-            set nodes [get_nodes -of_objects $net]
-            foreach node $nodes {
-                puts $fp "{"
-                    puts $fp "\"name\":\"$node\","
-                    puts $fp "\"cost_code\":\"[get_property COST_CODE $node]\","
-                    puts $fp "\"cost_code_name\":\"[get_property COST_CODE_NAME $node]\","
-                    puts $fp "\"speed_class\":\"[get_property SPEED_CLASS $node]\","
-                    puts $fp "\"wires\":\["
-                    set wires [get_wires -of_objects $node]
-                    foreach wire $wires {
-                        puts $fp "{"
-                            puts $fp "\"name\":\"$wire\","
-                            puts $fp "\"cost_code\":\"[get_property COST_CODE $wire]\","
-                            puts $fp "\"speed_index\":\"[get_property SPEED_INDEX $wire]\","
-                        puts $fp "},"
-                    }
-                    puts $fp "\],"
-                puts $fp "},"
-            }
-            puts $fp "\],"
-
-            set opin [get_pins -leaf -of_objects [get_nets $net] -filter {DIRECTION == OUT}]
-            puts $fp "\"opin\": {"
-                puts $fp "\"name\":\"$opin\","
-                set opin_site_pin [get_site_pins -of_objects $opin]
-                puts $fp "\"site_pin\":\"$opin_site_pin\","
-                puts $fp "\"site_pin_speed_index\":\"[get_property SPEED_INDEX $opin_site_pin]\","
-                puts $fp "\"node\":\"[get_nodes -of_objects $opin_site_pin]\","
-                puts $fp "\"wire\":\"[get_wires -of_objects [get_nodes -of_objects $opin_site_pin]]\","
-            puts $fp "},"
-            set ipins [get_pins -of_objects [get_nets $net] -filter {DIRECTION == IN} -leaf]
-            puts $fp "\"ipins\":\["
-            foreach ipin $ipins {
-                puts $fp "{"
-                    set delay [get_net_delays -interconnect_only -of_objects $net -to $ipin]
-                    puts $fp "\"name\":\"$ipin\","
-                    puts $fp "\"ic_delays\":{"
-                        foreach prop {"FAST_MAX" "FAST_MIN" "SLOW_MAX" "SLOW_MIN"} {
-                            puts $fp "\"$prop\":\"[get_property $prop $delay]\","
-                        }
-                    puts $fp "},"
-                    set ipin_site_pin [get_site_pin -of_objects $ipin]
-                    puts $fp "\"site_pin\":\"$ipin_site_pin\","
-                    puts $fp "\"site_pin_speed_index\":\"[get_property SPEED_INDEX $ipin_site_pin]\","
-                    puts $fp "\"node\":\"[get_nodes -of_objects $ipin_site_pin]\","
-                    puts $fp "\"wire\":\"[get_wires -of_objects [get_nodes -of_objects $ipin_site_pin]]\","
-                puts $fp "},"
-            }
-            puts $fp "\],"
-
-        puts $fp "},"
-    }
-
-    puts $fp "\]"
-    close $fp
-}
+source $::env(XRAY_UTILS_DIR)/write_timing_info.tcl
 
 proc create_design {design_name sig_mask verilogs} {
```

Comment:
Maybe it would be useful to include also additional tcl files (such as the one generated in arch-defs vivado target) to use this runme.tcl more generically, to produce the json5 timings file of an implemented design.

Reply:
In the case of implemented design, `runme.tcl` isn't required, just:
```
open_checkpoint <file>
source ${PATH_TO_UTILS}/write_timing_info.tcl
write_timing_info timing.json5
```

So I don't believe we need to modify `runme.tcl`